akka-actor-typed-tests/src/test/scala/akka/actor/typed/SupervisionSpec.scala

```diff
@@ -690,4 +690,58 @@ class SupervisionSpec extends ActorTestKit with TypedAkkaSpecWithShutdown {
     }
 
   }
+
+  val allStrategies = Seq(
+    SupervisorStrategy.stop,
+    SupervisorStrategy.restart,
+    SupervisorStrategy.resume,
+    SupervisorStrategy.restartWithBackoff(1.millis, 100.millis, 2D),
+    SupervisorStrategy.restartWithLimit(1, 100.millis)
+  )
+
+  allStrategies.foreach { strategy ⇒
+
+    s"Supervision with the strategy $strategy" should {
+
+      "that is initially stopped should be stopped" in {
+        val actor = spawn(
+          Behaviors.supervise(Behaviors.stopped[Command])
+            .onFailure(SupervisorStrategy.restart)
```

Comment:
do you not want to use `strategy` here?

Reply:
ups

Reply:
I have fixed this. No permission to push to your branch, so continued in https://github.com/akka/akka/pull/25334

Resume was missing.